src/7-to-8/major-changes/cylc-install.rst

```diff
@@ -114,6 +114,36 @@ use :cylc:conf:`flow.cylc[scheduler]install`.
 
 See :ref:`the user guide <RemoteInit>` for more details.
 
+.. warning::
+
+   If you have tasks that mirror to a remote platform (such as
```

Comment:
Maybe clarify "mirror" i.e. "transfer data".

The use of mirror is obvious in fcm_make cases, but might not be so obvious otherwise.